Maple2Storage/Helpers/StringUtils.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Maple2Storage.Helpers
+{
+    public class StringUtils
+    {
+        public static int[] SplitStringIntoInts(string list)
+        {
+            string[] split = list.Split(new char[1] {','});
+            List<int> numbers = new List<int>();
+
+            foreach (string n in split)
+            {
+                if (int.TryParse(n, out int parsed))
+                {
+                    numbers.Add(parsed);
+                }
+            }
+
+            return numbers.ToArray();
+        }
+    }
+}
```

Comment:
This entire method and thereby the class seems pretty unnecessary when you can do this with a simple one liner:
```c#
List<int> numbers = new List<int>(Array.ConvertAll(list.Split(','), int.Parse));
```